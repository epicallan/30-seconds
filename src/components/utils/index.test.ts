import * as prettyFormat from 'pretty-format';
import {getPagesData} from '.';

describe('utils', () => {
    it('should split data', () => {
        const data = [
            ['a', 'a'], ['c', 'c'],
            ['d', 'd'], ['e', 'e'],
            ['f', 'f'], ['g', 'g'],
            ['l', 'l'], ['k', 'k'],
        ];
        const results = getPagesData(data, 4);
        expect(prettyFormat(results)).toMatchSnapshot();
     });
});
