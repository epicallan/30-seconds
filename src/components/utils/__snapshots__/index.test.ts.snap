// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils should split data 1`] = `
"Array [
  Array [
    Array [
      \\"a\\",
      \\"a\\",
    ],
    Array [
      \\"c\\",
      \\"c\\",
    ],
    Array [
      \\"d\\",
      \\"d\\",
    ],
    Array [
      \\"e\\",
      \\"e\\",
    ],
  ],
  Array [
    Array [
      \\"f\\",
      \\"f\\",
    ],
    Array [
      \\"g\\",
      \\"g\\",
    ],
    Array [
      \\"l\\",
      \\"l\\",
    ],
    Array [
      \\"k\\",
      \\"k\\",
    ],
  ],
]"
`;
